docs/modules/ROOT/pages/map-configuration.adoc

```diff
@@ -0,0 +1,67 @@
+= Map Configuration
+:description: Hazelcast maps are used to distribute data across a cluster and make it easy to scale your applications while protecting data against member failures. You can create and configure maps using the `Map` resource.
+
+Hazelcast maps are used to distribute data across a cluster and make it easy to scale your applications while protecting data against member failures. You can create and configure maps using the `Map` resource.
+
+== Configuring Map Resource
+
+Below are the configuration options for the `Map` resource. You can find more detailed information in xref:api-ref.adoc#mapspec[API Reference] page.
+
+[cols="20%m,80%a"]
+|===
+|Property|Description
+
+|name
+|Name of the map config to be created. If empty, the CR name will be used. It cannot be updated after the map config is created successfully.
+
+|backupCount
+|Count of synchronous backups.
+
+|timeToLiveSeconds
+|Maximum time in seconds for each entry to stay in the map.
+
+|maxIdleSeconds
+|Maximum time in seconds for each entry to stay idle in the map.
+
+|eviction
+|Configuration for removing data from the map when it reaches its max size.
+
+- persistenceEnabled: Eviction policy to be applied when map reaches its max size according to the max size policy.
```

Comment:
I think you mean `evictionPolicy` instead of `persistanceEnabled`.
```suggestion
- evictionPolicy: Eviction policy to be applied when map reaches its max size according to the max size policy.
```